packages/crypto/src/encryption.ts

```diff
@@ -1,26 +1,32 @@
-import SHA256 from 'crypto-js/sha256';
 import AES from 'crypto-js/aes';
 import Utf8 from 'crypto-js/enc-utf8';
+import { compare, hash } from 'bcryptjs';
 
-const STRETCH_ROUNDS = 5000;
+export const SALT = '$2a$10$iLpsQfxfYc3FKZZft82qTe';
```

Comment:
We could use bcryptjs's `genSalt()` method, but that generates a random one each time. This makes it impossible to rehash to the same value to decrypt with. So, making a fixed salt. I got this from running `genSalt()` once locally.